packages/server/test/unit/project_spec.js

```diff
@@ -468,6 +469,28 @@ This option will not have an effect in Some-other-name. Tests that rely on web s
         expect(scaffold.plugins).not.to.be.called
       })
     })
+
```

Comment:
Is there a test to ensure the scaffold isn't called when in text terminal and without the env var? Should probably add one.